db/migrate/20230707031122_add_watched_precedence_over_muted_index_to_user_options.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class AddWatchedPrecedenceOverMutedIndexToUserOptions < ActiveRecord::Migration[7.0]
+  def change
+    add_index :user_options, %i[user_id watched_precedence_over_muted]
```

Comment:
I'm not sure if `user_id` is necessary here since the filter is only on the `watched_precedence_over_muted` column.